code/script.js

```diff
@@ -0,0 +1,170 @@
+const projectsContainer = document.getElementById("projects")
+const commitsContainer = document.getElementById("commits")
+const user = 'anndimi'
+const USER_URL = `https://api.github.com/users/${user}`
+const userContainer = document.getElementById("userProfile")
+
+const checkHTML = document.getElementById("HTML")
+const checkCSS = document.getElementById("CSS")
+const checkJavaScript = document.getElementById("JavaScript")
+let selectedLanguages = ['HTML', 'CSS', 'JavaScript']
+let chartDrawn = false
+
+//Create link images to social media
+const createImg = (url, alt) => {
+    let image = document.createElement("img")
+    image.src = url
+    image.alt = alt
+    return image
+}
+
+const linkedinIcon = createImg("./assets/linkedin-icon.png", "linkedin")
+const githubIcon = createImg("./assets/github-icon.png", "github")
+const facebookIcon = createImg("./assets/facebook-icon.png", "facebook")
+const branchIcon = createImg("./assets/branch-icon.png", "branch")
+const pushIcon = createImg("./assets/push-icon.png", "push")
+const repoIcon = createImg("./assets/repo-icon.png", "repo")
+
+let heroImgArray = ["./assets/img1.webp", "./assets/img2.webp", "./assets/img3.webp"],
+    seconds = 25
+
+//Loop through hero images.
+function heroImgSequence() {
+    window.clearTimeout();
+    for (let i = 0; i < heroImgArray.length; i++) {
+        setTimeout(function () {
+            const heroImg = document.getElementById("heroImg")
+            heroImg.style.background = "linear-gradient(rgba(44, 40, 27, 0.39), rgba(41, 37, 25, 0.76)),url(" + heroImgArray[i] + ")"
+            heroImg.style.backgroundPosition = "center"
+            heroImg.style.backgroundRepeat = "no-repeat"
+            heroImg.style.backgroundPosition = "relative"
+            heroImg.style.backgroundSize = "cover"
+            if ((i + 1) === heroImgArray.length) {
+                setTimeout(function () { heroImgSequence() }, (seconds * 1000))
+            } else { i++; }
+        }, (seconds * 1000) * i)
+    }
+}
+heroImgSequence()
+
+
+//Function to fetch and display profile information.
+const userProfile = () => {
+    fetch(USER_URL)
+        .then(res => res.json())
+        .then(data => {
+            console.log(data)
+            userContainer.innerHTML = `
+        <img src="${data.avatar_url}" class="avatar"/>
+        <p class="full-name">${data.name}</p>
+        <p class="username">Username: ${data.login}</p>
+        <p class="location">${data.location}</p>
+        <a href="${data.html_url}" target="blank"><img src="${githubIcon.src}" alt="${githubIcon.alt}"  class="github-icon"/></a>
+        <a href="${data.blog}" target="blank"><img src="${linkedinIcon.src}" alt="${linkedinIcon.alt}" class="linkedin-icon"/></a>
+        <a href="https://www.facebook.com/anna.dimitrakopoulos" target="blank"><img src="${facebookIcon.src}" alt="${facebookIcon.alt}" class="facebook-icon"/></a>
+        `
+        })
+}
+
+//Function to fetch and display my repos.
```

Comment:
Good that you wrote additional code in the function what each thing does! super clear